x11-packages/desktop-file-utils/hooks/postrm.in

```diff
@@ -1,6 +1,6 @@
 #!@TERMUX_PREFIX@/bin/sh
 
-if [ "${1}" = "remove" ]; then
+if [ "${1}" = "remove" ] || [ "@TERMUX_PACKAGE_FORMAT@" = "pacman" ]; then
```

Comment:
```suggestion
if [[ "${1}" = "remove" || "@TERMUX_PACKAGE_FORMAT@" = "pacman" ]]; then
```

Reply:
Also the whole file should be reindented to use tabs for consistency with other scripts.

Reply:
`sh` is used here, not `bash`. So it is not possible to use `[[`.

Reply:
It should be same as in `x11-packages/shared-mime-info/postrm`, with bash in shebang.